src/Admin/GeneralConventionAdmin.php

```diff
@@ -0,0 +1,248 @@
+
+                    $datagrid->setValue($property[0], null, $value);
+                },
+                'btn_add' => false,
+            ])
+            ->add('committeeZone', ModelAutocompleteType::class, [
+                'multiple' => false,
+                'label' => 'Comité (zone)',
+                'required' => false,
+                'property' => ['name', 'code'],
+                'callback' => function (AdminInterface $admin, array $property, $value): void {
+                    $datagrid = $admin->getDatagrid();
+                    $query = $datagrid->getQuery();
+                    $rootAlias = $query->getRootAlias();
+                    $query
+                        ->andWhere($rootAlias.'.type = :type_city')
+                        ->setParameter('type_city', Zone::CITY)
+                    ;
+
+                    $datagrid->setValue($property[0], null, $value);
+                },
+                'btn_add' => false,
+            ])
+            ->add('districtZone', ModelAutocompleteType::class, [
+                'multiple' => false,
+                'label' => 'Circonscription',
+                'required' => false,
+                'property' => ['name', 'code'],
+                'callback' => function (AdminInterface $admin, array $property, $value): void {
+                    $datagrid = $admin->getDatagrid();
+                    $query = $datagrid->getQuery();
+                    $rootAlias = $query->getRootAlias();
+                    $query
+                        ->andWhere($rootAlias.'.type = :type_district')
+                        ->setParameter('type_district', Zone::DISTRICT)
+                    ;
+
+                    $datagrid->setValue($property[0], null, $value);
+                },
+                'btn_add' => false,
+            ])
+            ->add('organizer', EnumType::class, [
+                'label' => 'Instance organisatrice',
+                'class' => OrganizerEnum::class,
+                'choice_label' => function (OrganizerEnum $organizer) {
+                    return 'general_convention.organizer.'.$organizer->value;
+                },
+            ])
+            ->add('reporter', ModelAutocompleteType::class, [
+                'label' => 'Auteur de la remontée',
+                'required' => true,
+                'minimum_input_length' => 1,
+                'items_per_page' => 20,
+                'property' => [
+                    'search',
+                ],
+                'to_string_callback' => static function (Adherent $adherent): string {
+                    return \sprintf(
+                        '%s (%s) [%s]',
+                        $adherent->getFullName(),
+                        $adherent->getEmailAddress(),
+                        $adherent->getId()
+                    );
+                },
+                'btn_add' => false,
+            ])
+            ->add('reportedAt', null, [
+                'label' => 'Date de la remontée',
```

Comment:
it's more userfriendly with this option, wdyt ?
```suggestion
                'label' => 'Date de la remontée',
                'widget' => 'single_text',
```